pkg/metrics/metrics_controller.go

```diff
@@ -260,13 +260,15 @@ func (mc *Controller) Run(healthChan chan<- *healthcheck.ControllerHeartbeat, st
 	DefaultRegisterer.MustRegister(BuildInfo)
 	DefaultRegisterer.MustRegister(ControllerIpvsMetricsExportTime)
 
+	mux := &http.ServeMux{}
```

Comment:
Small nit here, can you change this to `http.NewServeMux()` instead? It amounts to the same, but seems better to use the default function provided.